docs/api/instruments/advantest/advantestR624X.rst

```diff
@@ -0,0 +1,412 @@
+#########################################################
+Advantest R6245/R6246 DC Voltage/Current Sources/Monitors
+#########################################################
+.. currentmodule:: pymeasure.instruments.advantest.advantestR624X
+
+.. contents::
+
+**********************************************
+General Information
+**********************************************
+
+The R6245/6246 Series are DC voltage/current sources and monitors having
+source measurement units (SMUs) with 2 isolated channels. The series covers
+wide source and measurement ranges. It is ideal for measurement of DC characteristics
+of items ranging from separate semiconductors such as bipolar transistors, MOSFETs
+and GaAsFETs, to ICs and power devices. Further, due to the increased measuring speed
+and synchronized 2-channel measurement function, device I/O characteristics can be
+measured with precise timing at high speed which previously difficult to accomplish.
+Due to features such as the trigger link function and the sequence programming function
+which automatically performs a series of evaluation tests automatically, the R6245/6246
+enable much more efficient evaluation tests.
+
+There is a total of 99 commands, the majority of commands have been implemented. Device
+documentation is in Japanese, and the device options are enormous. The implementation is
+based on 6245S-GPIB-B-FHJ-8335160E01.pdf, which can be downloaded from the ADCMT website.
+
+Command Translation
+===================
+Alphabetical list of implemented R6245/R6246 commands and their corresponding
+method/attribute names in this instrument driver.
```

Comment:
I would not include this table at all -- it's a manual list of the implemented API, and as such is bound to go out of sync with the implementation itself. 
What use does this fulfill that the autogenerated API reference at the bottom does not do (better)?
Is it really important/useful to users/consumers of this documentation to know the raw wire commands behind the implemented API? Much of the purpose of pymeasure to abstract _away_ that information...

Reply:
> Much of the purpose of pymeasure to abstract away that information...
Agree, but while owning this device you get the urge to look at the 500 page manual and translate some BASIC examples to pymeasure. Thus I found it useful, myself.

Reply:
Removed the table, although same table is in Agilent B1500 documentation, that one gave me the idea.